prac_07/project.py

```diff
@@ -32,8 +32,10 @@ def __ge__(self, other):
     def __lt__(self, other):
         return self.start_date < other.start_date
 
-    def update_priority(self, priority):
-        self.priority = priority
-
     def update_completion(self, completion):
```

Comment:
These "update_x" functions are unnecessary. Just set the value, like:
`project.priority = 1`